src/vm/eetwain.cpp

```diff
@@ -2404,14 +2404,16 @@ unsigned scanArgRegTable(PTR_CBYTE    table,
    Returns size of things pushed on the stack for ESP frames
 
    Arguments:
-      table   - The pointer table
-      curOffs - The current code offset
-      info    - Incoming arg used to determine if there's a frame, and to save results
+      table       - The pointer table
+      curOffsRegs - The current code offset that should be used for reporting registers
+      curOffsArgs - The current code offset that should be used for reporting args
```

Comment:
I added a comment before the assert that references the comment in EnumGCRefs.

Reply:
Maybe it would be better that instead of two offset arguments to have just one offset and a flag requiring the register offset to be adjusted (e.g. `inactiveFrame`, `callInProgress` etc.). That would avoid that need for asserts such as "one is less than the other".

Reply:
I started with that version but decided this is better. This method is called from several places and some of them don't really care about the registers and don't have the active/inactive frame info. It seemed better to just pass the same curOffs from those callers.